packages/manager-components/src/components/ManagerText/ManagerText.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { PropsWithChildren } from 'react';
+import { OsdsText } from '@ovhcloud/ods-components/react';
+import { ODS_TEXT_LEVEL, ODS_TEXT_SIZE } from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import './translations';
+
+import { useIsAuthorized } from '../../hooks/iam';
+
+export type ManagerTextProps = PropsWithChildren<{
+  isIamAuthorized?: boolean;
+  action?: string;
```

Comment:
**nitpick:** Considering if we implement any code-annotations, it would be nice to rename it to iamAction to easily identify the actions only related to iam.

Reply:
👍